packages/labs/gen-utils/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "private": true,
+  "name": "@lit-labs/gen-utils",
+  "description": "Utilities for lit code generators",
+  "version": "0.1.0",
```

Comment:
I think if you check in 0.0.0, then changesets will bump this correctly to 0.1.0.

Reply:
Done